ProcessLib/LIE/Common/Utils.h

```diff
@@ -18,11 +18,28 @@ namespace ProcessLib
 namespace LIE
 {
 
-/// compute a normal vector of the given element
-void computeNormalVector(MeshLib::Element const& e, Eigen::Vector3d & normal_vector);
-
-/// compute a rotation matrix from global to local coordinates based on the given normal vector
-void computeRotationMatrix(Eigen::Vector3d const& normal_vector, int dim, Eigen::MatrixXd &matR);
+/// Compute a normal vector of the given element
+///
+/// Computed normal vector is oriented in the left direction of the given line
+/// element such that computeRotationMatrix2D() returns the indentity matrix for
+/// line elements parallel to a vector (1,0,0)
+void computeNormalVector(MeshLib::Element const& e, unsigned const global_dim,
```

Comment:
I tried to move it to MeshLib, but the code (due to global dimension checks) is too specific for the current applications and needs more changes before it can go into MeshLib.